lib/OBSWebSocket.js

```diff
@@ -37,20 +35,20 @@ class OBSWebSocket extends Socket {
       const messageId = args['message-id'] = generateMessageId(); // eslint-disable-line no-multi-assign
 
       // Submit the request to the websocket.
-      log.debug('[Core:send]', messageId, requestType, args);
+      debug('[send] %s %s %O', messageId, requestType, args);
       this._socket.send(JSON.stringify(args));
```

Comment:
Yeah, I had it that way originally. You had to construct the object, call connect, then check getAuthRequired, then call authenticate. Not at all opposed to going back to that or something similar. Might be able to hide the getAuthRequired call in one of those.